src/tribler/core/utilities/tracker_utils.py

```diff
@@ -62,7 +62,7 @@ def get_uniformed_tracker_url(tracker_url):
         # accessing urlparse attributes may throw UnicodeError's or ValueError's
         try:
             # scheme must be either UDP or HTTP
-            if url.scheme == 'udp' or url.scheme == 'http':
+            if url.scheme == 'udp' or url.scheme == 'http' or url.scheme == 'https':
```

Comment:
**nitpick:** I'd simplify it to:
```suggestion
            if url.scheme in ('udp', 'http', 'https'):
```

Reply:
If I understood correctly, tuples in Python have O(n) complexity for checking an item's presence in the list.
Sets, however, have O(1) complexity.

So, I propose to use a set here:
```python
            if url.scheme in {'udp', 'http', 'https'}:
```

Reply:
I used to use tuples because some time ago, Python was able to cache and re-use immutable tuple literals but re-constructed mutable set literals each time the line of the code was executed (and that was much slower), but now Python can properly cache mutable set literals as well, and using set literals is better now.

Reply:
Good point! addressed.